lib/v2/screens/profile_screens/guardians/invite__guardians_sent/invite_guardians_sent_screen.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/i18n/guardians.i18n.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/v2/navigation/navigation_service.dart';
+import 'package:seeds/widgets/main_button.dart';
+
+class InviteGuardiansSentScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          leading: const SizedBox.shrink(),
+        ),
+        body: WillPopScope(
+          onWillPop: () async {
+            Navigator.popUntil(context, ModalRoute.withName('/'));
```

Comment:
This is not working, 
@RaulUrtecho 

Its navigating to the security screen but not to the guardians tabs.

Reply:
I added this will pop because i want to navigate to the guardians tabs when the user taps device back button. 
Same behavior when they tap the OK button

Reply:
try:
```
          onWillPop: () async {
            Navigator.popUntil(context, ModalRoute.withName(Routes.guardianTabs));
            return true;
          },
```

Reply:
`Navigator.popUntil(context, ModalRoute.withName(Routes.guardianTabs));`

This crashes the app, seems to put it into an infinite loop

Reply:
That is strange to me it worked without any loop or crash. 
By the way, is it okay, can you request my myself?

<img src="https://user-images.githubusercontent.com/42857405/120134014-db167480-c192-11eb-9a82-ee76481b1fc8.gif" width="300">

Reply:
weeeirddd, ill try again. 
And no, we should add checks to stop someone from requesting oneself

Reply:
done
Second issue follow up prs